crates/sui-core/src/unit_tests/authority_tests.rs

```diff
@@ -124,40 +124,69 @@ fn compare_transaction_info_responses(
 }
 
 async fn construct_shared_object_transaction_with_sequence_number(
```

Comment:
Not related to your change, but could you add a TODO here to cleanup this function? It's doing a lot more than the function name says.